processing/src/test/java/io/druid/query/ChainedExecutionQueryRunnerTest.java

```diff
@@ -154,7 +155,7 @@ public void run()
     EasyMock.verify(watcher);
   }
 
-  @Test
+  @Test(timeout = 60000)
```

Comment:
One thing I've seen in the past is having a fudge factor that can globably be used to increase and decrease time outs based on the power of the machine and/or cluster (usually manually set on a machine basis but could possibly be dirived from processor and/or family). It's ussually multiplied by what ever the default timeout for each test.